tests/models/test_portfolio_role.py

```diff
@@ -291,3 +293,50 @@ def test_does_not_have_permission_set():
     assert not port_role.has_permission_set(
         PermissionSets.EDIT_PORTFOLIO_APPLICATION_MANAGEMENT
     )
+
+
+def test__repr__():
+    perm_sets = PermissionSets.get_many(
+        [PermissionSets.VIEW_PORTFOLIO_FUNDING, PermissionSets.VIEW_PORTFOLIO_REPORTS]
+    )
+    port_role = PortfolioRoleFactory.create(permission_sets=perm_sets)
+    assert (
+        str(port_role).startswith("<PortfolioRole") is True
+    ), "PortfolioRole default description start with <PortfolioRole"
```

Comment:
```suggestion
    assert (
        repr(port_role).startswith("<PortfolioRole") is True
    ), "PortfolioRole default description start with <PortfolioRole"
```

You can't guarantee that `__str__` calls `__repr__` and we don't want to require that behavior. We also should probably check that:

```
for perm_set in perm_sets:
  assert perm_set.name in repr(port_role)
```

or something like that.

Reply:
jum, good to know.

Reply:
@kylelaker-ccpo Had to look this up.  I didn't know there were two ways to get a string representation.

Reply:
Oh I see, PortfolioRole class implements __repr__ directly and includes the tested string.